build/yaml/steps/codeanalysis/suppress.gdnsuppress

```diff
@@ -8,6 +8,326 @@
     }
   },
   "results": {
+    "314cea1b6deefd251bf13ecf10566ae3db4700f6b21ae4fe130fabed17849154": {
+      "signature": "314cea1b6deefd251bf13ecf10566ae3db4700f6b21ae4fe130fabed17849154",
+      "alternativeSignatures": [
+        "74fbc162fb33461daf572e28d61feee2144e1c74980f5ba1b8d9ec65996e76df"
+      ],
+      "target": "binaries-windows-Release/x64/CommonLibTests.exe",
+      "memberOf": [
+        "BinSkimSuppression"
+      ],
+      "tool": "binskim",
+      "ruleId": "BA2004",
+      "justification": "linking to libs we cannot change.",
```

Comment:
Can you be more specific in the justification about which libs are causing BA2004?

Reply:
Sure